CLIENT-SPECIFICATION.md

```diff
@@ -71,6 +71,18 @@ becomes this:
 git-checkout
 ```
 
+Page names MAY contain mix capitalization - such page names MUST be transparently lowercased. For example, the page name:
```

Comment:
Is this definitely correct?  Both version sound correct to me and I can't make up my mind.  I'm slightly inclined towards 'mixed'.
```suggestion
Page names MAY contain mixed capitalization - such page names MUST be transparently lowercased. For example, the page name:
```